### PR TITLE
Added DigitalOcean provider

### PR DESCRIPTION
**Pull request by mandalae**
I've added assets and a provider for DigitalOcean.

I tried to follow the development and contribution guides, but if I've done anything wrong, please let me know.

### CODE REVIEW

**Comment by mandalae** [comment]
@mingrammer Is there any way to get this reviewed? I notice that there are quite a lot of PRs outstanding, so I'm wondering if this is still actively being worked on?

File: config.py [hunk 1]
```diff
@@ -59,6 +60,7 @@
 UPPER_WORDS = {
     "aws": ("aws", "api", "ebs", "ec2", "efs", "emr", "rds", "ml", "mq", "nat", "vpc", "waf", "sdk"),
     "azure": ("ad", "b2c", "ai", "api", "cdn", "ddos", "dns", "fxt", "hana", "hd", "id", "sap", "sql", "vm"),
+    "azure": ("vpn", "vpc"),
```
Comment (mingrammer) [inline config.py:63]: "azure" key is duplicated.

**Review by mingrammer — CHANGES_REQUESTED** [review]
Plz check the azure config.

**Comment by mingrammer** [comment]
Sorry for the late reply. @mandalae

File: config.py (hunk 1 above)
Comment (mandalae) [reply config.py]: Fixed :)

**Comment by mandalae** [comment]
Thanks for reviewing, I've fixed that comment now :)

**Comment by mingrammer** [comment]
@mandalae Please re-run the autogen script!

**Comment by mandalae** [comment]
@mingrammer  Done, a lot of the changes in this PR came from doing that in the first place I think. I only really added the DO stuff, I never meant to change anything in AWS or Azure.

**Review by mingrammer — APPROVED** [review]
LGTM!. Thank you for working on it.

**Comment by mingrammer** [comment]
v0.21.0 is now released.

**Comment by dajor** [comment]
Is in the last version not a __init__.py missing for DigitalOcean?

**Comment by mandalae** [comment]
Correct @dajor I've added it in this PR: https://github.com/mingrammer/diagrams/pull/646 :) - Thanks for pointing it out.

**Comment by dajor** [comment]
There is one more bug:

from diagrams import Diagram

from diagrams.digitalocean.network import LoadBalancer
from diagrams.digitalocean.storage import Space

from diagrams.digitalocean.compute import Containers


with Diagram("Test", show=False):
    LoadBalancer >> Containers >> Space


Traceback (most recent call last):
    Containers >> Space
TypeError: unsupported operand type(s) for >>: 'type' and 'type'

**Pull request merged**